### PR TITLE
Sets background color of selected rows while drag&dropping

### PR DESCRIPTION
**Pull request by yuvallevanoni**
- to stay the same

### CODE REVIEW

File: src/components/common-v2/SortableTable/index.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ const rowProps = (args: any) => ({
   index: args.rowIndex,
 });
```
Comment (yuvallevanoni) [inline src/components/common-v2/SortableTable/index.tsx:29]: The same logic exists in the old WMTable

File: src/components/Screen-v2/CoursesScreen/style.module.scss [hunk 2]
```diff
@@ -177,4 +177,13 @@
       background-color: var(--white);
     }
   }
+
```
Comment (yuvallevanoni) [inline src/components/Screen-v2/CoursesScreen/style.module.scss:180]: selected row will not change background while drag&drop

File: src/components/Screen-v2/CoursesScreen/style.module.scss [hunk 3]
```diff
@@ -201,20 +201,25 @@
...
+            overflow: visible !important; // using important to avoid inline-styles
+            background-color: inherit;
 
-          .actionsCell {
-            visibility: visible;
-            opacity: 1;
+            .actionsCell {
+              visibility: visible;
+              opacity: 1;
+            }
```
Comment (nunibaranes) [inline src/components/Screen-v2/CoursesScreen/style.module.scss:222]: Please add styles for selectedRow while dragging.

**Review by nunibaranes — CHANGES_REQUESTED** [review]
Looks great, please see my comment :)

File: src/components/Screen-v2/CoursesScreen/style.module.scss (hunk 3 above)
Comment (yuvallevanoni) [reply src/components/Screen-v2/CoursesScreen/style.module.scss:222]: in row 328

**Review by nunibaranes — APPROVED** [review]
Good job :) 
Approved!

**Pull request merged**